page_objects/drivers/local_browsers.py

```diff
@@ -72,3 +72,31 @@ def headless_chrome():
         local_driver = webdriver.Chrome(options=options)
 
         return local_driver
+
+    def app_details(self, desired_capabilities, app_package, app_activity):
```

Comment:
Can we have the `Mobile` driver setup methods in a separate object instead of `LocalBrowsers` ?

Reply:
As per our discussion, renamed file name to local options